doc/md/meet-motoko.md

```diff
@@ -2,7 +2,106 @@
 sidebar_position: 1
 ---
 
-# Meet Motoko: The language shaping the future of Web3
+# Meet Motoko: The language of the Internet Computer
 
-Hello, world example
-Brief description of what Motoko is / key design aspects
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+import Tabs from "@theme/Tabs";
+
+import TabItem from "@theme/TabItem";
+
+<MarkdownChipRow labels={["Beginner", "Getting started"]} />
+
+<img src="https://github.com/user-attachments/assets/844ca364-4d71-42b3-aaec-4a6c3509ee2e" alt="Logo" width="150" height="150" />
+
+## Introduction
+
+Motoko is the native programming language of the Internet Computer. It’s a modern, **actor-based** language designed for safety, efficiency, and scalability. With native support for asynchronous messaging, WebAssembly (Wasm) interoperability, robust type safety, and data persistence, Motoko makes it easy to build secure and performant decentralized web applications.
```

Comment:
```suggestion
Motoko is the native programming language of the Internet Computer. It’s a modern, **actor-based** language designed for safety, efficiency, and scalability. With native support for asynchronous messaging, Candid interoperability, robust type safety, and data persistence, Motoko makes it easy to build secure and performant decentralized web applications.
```
Might be better. Wasm interop tends to sugges the wasm component model which neither Motoko nor the IC supports yet.